tools/src/main/scala/caliban/tools/ClientWriter.scala

```diff
@@ -380,6 +380,10 @@ object ClientWriter {
       .map(v => s"""case ${typedef.name}.${safeEnumValue(v.enumValue)} => __EnumValue("${v.enumValue}")""")
       .mkString("\n")}
           }
+
+          val values: Vector[${enumName}] = Vector(${typedef.enumValuesDefinition
```

Comment:
How about a `Set` instead of `Vector`?

Reply:
It's possible to go from `Vector` to `Set` but not the other way around, I believe this is why existing enumeration implementations expose their members as sequences (scala enumerations and scalapb being two examples)

Reply:
I agree with you that people _shouldn't_ depend on the ordering of members of an enumeration, but preventing them from doing so at all is just waiting for someone to come and ask for that feature, then breaking all consumers who are already using whatever implementation we decide on here.

Reply:
Yeah, I don't think it's right to expect any kind of order but I don't have strong feelings about it. Fine to merge like this.